src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/ManageLocationsTest.kt

```diff
@@ -28,7 +31,7 @@ internal class ManageLocationsTest : IntegrationTest() {
 
   @Test
   @Order(1)
-  fun `map missing location name and location type to agency id FROM_AGENCY`() {
+  fun `map missing location name and location type to agency id STOPOVER_AGENCY`() {
```

Comment:
This was an attempt to see if just having one option in the choice of links would fix the CI but alas it did not.